bugsnag/__init__.py

```diff
@@ -1,10 +1,19 @@
 import sys
-import traceback
+import logging
 
 from bugsnag.configuration import Configuration, RequestConfiguration
 from bugsnag.notification import Notification
 
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stderr)
+formatter = logging.Formatter(
+    '%(asctime)s - [%(name)s] %(levelname)s - %(message)s')
```

Comment:
This format now includes the time and level. The time can be removed if you want, but it might be useful to know.

```
2016-08-20 15:06:38,284 - [bugsnag] WARNING - No API key configured, couldn't notify
```